website/content/en/docs/deployment/pipeline-profiles.md

```diff
@@ -0,0 +1,119 @@
+
+The `default-editor` SA needs to be annotated with an IAM role with sufficient permissions to access your S3 Bucket to run your pipelines. In the below steps we will be configuring a profile an IAM role with restricted access to a specific S3 Bucket using the `AwsIamForServiceAccount` plugin for Profiles. To learn more about Profiles read the [Profiles component guide]({{< ref "/docs/component-guides/profiles.md" >}}).
+
+>Note: If you choose to run your pipeline with a service account other than the default which is `default-editor`, you must make sure to annotate that service account with an IAM role with sufficient S3 permissions.
+
+#### Admin considerations
+
+- Kubeflow admins will need to create an IAM role for each Profile with the desired scoped permissions.
+- A `default-editor` SA exists in every Profile's namespace and will be annotated with the role ARN created for the profile. Pods annotated with the SA name will be granted the Profile role permissions.
+- The `default-editor` SA is used by various services in Kubeflow to launch resources in Profile namespaces. However, not all services do this by default.
+
+## Create a Profile
+
+After installing Kubeflow on AWS with one of the available [deployment options]({{< ref "/docs/deployment" >}}), you can configure Kubeflow Profiles with the following steps:
+
+1. Define the following environment variables:
+
+   ```bash
+   export CLUSTER_NAME=<your cluster name>
+   export CLUSTER_REGION=<your region>
+   export S3_BUCKET=<your s3 bucket>
+   export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+   export PROFILE_NAME=<the name of the profile to be created>
+   ```
+2. Retrieve IAM OIDC associated with your cluster.
+
+   ```bash
+   aws --region $CLUSTER_REGION eks update-kubeconfig --name $CLUSTER_NAME
+
+   export OIDC_URL=$(aws eks describe-cluster --region $CLUSTER_REGION --name $CLUSTER_NAME  --query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+   ```
+
+3. Create an IAM trust policy to authorize federated requests from the OIDC provider.
+
+   ```bash
+
+   cat <<EOF > trust.json
+   {
+   "Version": "2012-10-17",
+   "Statement": [
+       {
+       "Effect": "Allow",
+       "Principal": {
+           "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_URL}"
+       },
+       "Action": "sts:AssumeRoleWithWebIdentity",
+       "Condition": {
+           "StringEquals": {
```

Comment:
this does not have access to SA in profile namespace? we are using annotateOnly = true below